inc/planning.class.php

```diff
@@ -931,6 +931,11 @@ class='".$filter_data['type']."'>";
             echo "<li><a target='_blank' href='webcal://".$url['host'].":$port".
                  (isset($url['path'])?$url['path']:'')."$cal_url'>".
                  _sx("button", "Export")." - ".__("Webcal")."</a></li>";
+
+            echo "<li><a target='_blank' href='".$CFG_GLPI['root_doc'].
+                 "/front/planningcsv.php?uID=".$loginUser->fields['id']."'>".
```

Comment:
uID/gID params should correspond to selected user/group
```suggestion
                 "/front/planningcsv.php?uID=".$uID."&gID=".$gID."'>".
```